recipe/activate.sh

```diff
@@ -6,20 +6,24 @@ ERROR=false
 [[ "@cross_target_platform@" == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
 [[ "@cross_target_platform@" == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
 
+CUDA_CFLAGS=""
+CUDA_LDFLAGS=""
 if [ "${CONDA_BUILD:-0}" = "1" ]; then
-    CUDA_INCLUDE_DIR="${PREFIX}/${targetsDir}/include"
+    CUDA_CFLAGS="${CUDA_CFLAGS} -I${PREFIX}/${targetsDir}/include"
+    CUDA_CFLAGS="${CUDA_CFLAGS} -I${BUILD_PREFIX}/${targetsDir}/include"
+    CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${PREFIX}/${targetsDir}/lib/stubs"
+    CUDA_LDFLAGS="${CUDA_LDFLAGS} -L${BUILD_PREFIX}/${targetsDir}/lib/stubs"
```

Comment:
Note that we search both `$PREFIX` and `$BUILD_PREFIX` for both headers and stub libraries (preferring `$PREFIX`, which is the `host` environment)

This comes up in the tests as well since there it uses this code path there (as `CONDA_BUILD=1` in the tests). However in the test case `$PREFIX` is all that matters (as `$BUILD_PREFIX` refers to a directory that is no longer relevant)

Reply:
I think when cross compiling, we cannot use the stubs from host, only from build, otherwise we'd get a linker error. But, AFAIK the linker would keep searching, so if build is after host, we are fine.